README.md

```diff
@@ -9,11 +9,17 @@ A simple ray tracer written in pure CMake. Inspired by [raytracer.hpp](https://g
 The ray tracer writes its output to `stderr`, so you can use it with:
 
 ```shell
-cmake . -Dimage_width=64 -Dimage_height=64 -Dnum_procs=4 2> image.ppm
+cmake -Dimage_width=64 -Dimage_height=64 -Dnum_procs=4 -P CMakeLists.txt 2> image.ppm
 ```
 
 Which writes the output to `image.ppm`. Then use an image viewer capable of opening PPM files (or [this](http://www.cs.rhodes.edu/welshc/COMP141_F16/ppmReader.html)) to view.
 
+Alternatively, the ray tracer can directly write PNG files (PowerShell required for now). Use the `use_png` variable to set the name of the PNG file without the extension. The example below will write output to `render.png`.
```

Comment:
Feel free to give yourself some credit here (write something like 'courtesy of @benmcmorran' or 'this feature was implemented by @benmcmorran')

Reply:
Added, thanks!